graphql/schema.graphql

```diff
@@ -100,6 +114,7 @@ type Patch {
   duration: PatchDuration
   time: PatchTime
   taskCount: Int
+  moduleCodeChanges: [ModuleCodeChange]
```

Comment:
This will probably be easier to use if the contents are non-nullable. Also potentially the field itself as well (it is valid to have a patch with no code changes)